bitnami/mongodb/values.yaml

```diff
@@ -143,6 +146,14 @@ auth:
   ## @param auth.password MongoDB&reg; custom user password
   ## @param auth.database MongoDB&reg; custom database
   ##
+  # username: username
+  # password: password
+  # database: database
+  ## By default custom user role is readWrite to database configured above
+  # userRole: readWrite
+  #vaultCredentials: /tmp/credentialFile
+  ## Key used for replica set authentication
+  ## Ignored when mongodb.architecture=standalone
```

Comment:
Did you accidentally add these comments? I mean, the unique new parameter I can see here is `auth. vaultCredentials`

Reply:
vaultCredentials is the configuration holding the file path which is mounted by the vault. This is added as a comment to provide an example, on how to use it.

Reply:
Then we should make it very clear that's an example. Following the format
```
## @param {{PARAM_NAME}} {{PARAM_DESCRIPTION}}
## e.g:
## {{PARAM_NAME}}:
##   ...
##   ...
##
```
You can use this example as reference: https://github.com/bitnami/charts/blob/master/bitnami/mongodb/values.yaml#L1000